databricks_template_schema.json

```diff
@@ -1,12 +1,13 @@
 {
   "welcome_message": "Welcome to MLOps Stacks. For detailed information on project generation, see the README at https://github.com/databricks/mlops-stacks/blob/main/README.md.",
   "min_databricks_cli_version": "v0.210.1",
+  "min_databricks_cli_version": "v0.210.1",
```

Comment:
Why are we adding this again? Please remove

Reply:
done!